### ISSUE TITLE
Meteor 1.4 Support?

### ISSUE THREAD

**Issue by Ryuzavi** [meteorhacks/meteord#109]
Any chance this docker container will support Meteor 1.4?

**Comment by ghost** [comment]
@Ryuzavi, I am using https://hub.docker.com/r/abernix/meteord/ for now

**Comment by Ryuzavi** [comment]
@jacquesblom404 Yeah I'm also using an alternate atm.  Would be good to see this one get updated though as it's more trusted.

**Comment by vuhrmeister** [comment]
Does anyone know what need to be changed? I can't use this image since we have our own build process. So far I just updated the node version in the build scripts.

**Comment by arunoda** [comment]
This container won't have the support for 1.4 due to legacy compatibility.
There'll be a new repo: https://github.com/kadirahq/meteord
